sgp4/tests.py

```diff
@@ -84,6 +85,33 @@ def test_satrec_attributes(self):
         self.assertEqual(sat.mo, 0.3373093125574321)
         self.assertEqual(sat.no_kozai, 0.04722944544077857)
 
+    def test_satrec_sgp4init_attributes(self):
```

Comment:
Nice! I like that you've added a test that does a simple round-trip of the new method.

1. Is there any way to test if the epoch got set too, or is that difficult to test the way the object is designed?
2. This is minor, but: could the attributes go in the same order in the list of attributes and the list of checks? Otherwise it requires a bit of visual searching to see if the values match up in the test.

Reply:
(1) Done and (2) done.